ballot/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.conf.urls import patterns, url
+from django.contrib import admin
+
+from . import views
+
+admin.autodiscover()
+
+urlpatterns = patterns(
+    '',
+    url(r'^measure/(?P<measure_id>[0-9]+)$', views.measure_view),
+    url(r'^ballot/$', views.ballot_view))
```

Comment:
I see. I didn't understand from the frontend code the full URL that would be sent to the backend. Is there a good way to eyeball that?

Glad to make this change. It means I probably screwed up most of the URLs, as I wasn't understanding the frontend code well enough, and didn't know how to trigger them to hit the backend.

Thanks!